tests/server/listPuzzles.test.js

```diff
@@ -25,18 +25,18 @@ jest.mock("../../server/middleware", () => {
  * @returns nothing
  */
 function dataHelper(k) {
-  for (let i = 1; i <= k; i++) {
-    db.run(
-      `INSERT INTO Puzzle VALUES (${i}, "test puzzle ${i}", "test description ${i}")`
-    );
-  }
+  db.run(
+    `INSERT INTO Puzzle VALUES (1, "test puzzle 1", "test description 1")`
+  );
+
+  db.run(`INSERT INTO User_Puzzle VALUES (1, 1, NULL, 1)`)
 }
 
 // block console logging, generate mock data
 beforeAll(() => {
-  jest.spyOn(console, "log").mockImplementation(() => {});
-  jest.spyOn(console, "error").mockImplementation(() => {});
-  dataHelper(5);
+  jest.spyOn(console, "log").mockImplementation(() => { });
```

Comment:
you can just do 
```js
jest.spyOn(console, "log").mockImplementation();
jest.spyOn(console, "error").mockImplementation();
```
(but honestly we'll catch it later in the rework story so who gives a fuck)

Reply:
I'll just leave this and deal with it in the other one. As long as these tests pass, that's all that matters for this user story imo